### DISCUSSION TITLE
push access denied

### DISCUSSION THREAD

**Discussion by ealalin** [community/community#78486] [New to GitHub] [answered]
/git config --global ealalin.alain
[I m logged into github with ealalin.docx](https://github.com/community/community/files/13625596/I.m.logged.into.github.with.ealalin.docx)

**Comment by ft10101** [comment]
try using auth tokens

**Reply by ealalin** [reply, marked as answer]
I have created one ssh key

**Reply by ft10101** [reply]
Ensure that your SSH key or credentials are correctly set up and that you have the necessary permissions to push to the repository.

**Reply by ft10101** [reply]
if you found my reply helpful kindly mark it as an answer

**Comment by gabi-dev-tech** [comment]
After several days I was able to recover my Github account. Now I can authenticate and enter the platform. The problem is when I want to create a new repository. In the step of pushing to my "main" branch I get a message: denied to GabrielSuarezDev. I have already configured my remote settings again for the new password and the problem still persists. I am contacting you because I have made several attempts without obtaining a solution. Could you be so kind as to give me more information about it? I think that after the last suspension of my account I started having this problem.

**Comment by ealalin** [comment]
Gabriel , I havé a similar problème, still not résolved. I can tell you what I havé try.
But for this I havé to log to my OWN pc. I will do it and answer to you (sorry for the syntax but  I use thé keyboard of my I phone)

Rgds. Ealalin

Verstuurd vanaf mijn iPhone

Op 12 dec 2023 om 21:50 heeft Gabriel Suarez ***@***.***> het volgende geschreven:

﻿

After several days I was able to recover my Github account. Now I can authenticate and enter the platform. The problem is when I want to create a new repository. In the step of pushing to my "main" branch I get a message: denied to GabrielSuarezDev. I have already configured my remote settings again for the new password and the problem still persists. I am contacting you because I have made several attempts without obtaining a solution. Could you be so kind as to give me more information about it? I think that after the last suspension of my account I started having this problem.

**Comment by ealalin** [comment]
Gabriel ;

find my problem and find what I have try untill now:

Problem with the push:


(ll_env) C:\learning_log>git push -u origin main

remote: Permission to ealalin/learning_log.git denied to Ealalin02.

fatal: unable to access 'https://github.com/ealalin/learning_log.git/': The requested URL returned error: 403

Why ealalin02 .... it's the question that I have not yet resolved search  because I push into ealalin

solution that I have try:

Remove the Repository on GitHub

 Recreate the Repository

Initialize a New Local Repository

After all that the probleme is still the same

Another tryal : make a ssh key locally and put this key into GITHUB

Generate een ssh key



(ll_env) c:\learning_log>ssh-keygen -t edxxxxx -C ***@***.***"    " with edxxxxx is modified for security raison"

Generating public/private edxxxxx key pair.

Enter file in which to save the key (C:\Users\EALALIN/.ssh/id_edxxxxx): public

You look on your c:\users\userid\.ssh  directory   and you edit your  file .pub   copy the information from it and paas it to Github
On github

  1.  In the "Access" section of the sidebar, click  SSH and GPG keys.
  2.  Click New SSH key or Add SSH key.
  3.  In the "Title" field, add a descriptive label for the new key. For example, if you're using a personal laptop, you might call this key "Personal laptop".
  4.  Select the type of key, either authentication or signing. For more information about commit signing, see "About commit signature verification<https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification>."
  5.  In the "Key" field, paste your public key.
  6.  Click Add SSH key.
  7.  If prompted, confirm access to your account on GitHub. For more information, see "Sudo <https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/sudo-mode>

[cid:0a161e74-e15c-4c9d-abeb-04747c1ee9ea]

copy the key to your github => setting => ssh =>  key ( pass the info coppied) => then choise add key  and put your password

Check locally if the key is working fine

C:\learning_log>ssh -T ***@***.***

C:\learning_log>ssh -T ***@***.***

The authenticity of host 'github.com (ip adress)' can't be established.

ECDSA key fingerprint is SHA256:"the code of your key. "

Are you sure you want to continue connecting (yes/no/[fingerprint])? yes

Warning: Permanently added 'github.com,ip adresse' (ECDSA) to the list of known hosts.

Hi ealalin! You've successfully authenticated, but GitHub does not provide shell access.

 Retrying the push   : same problem

Retscheck the information locally

C:\learning_log>git config --global --lis

***@***.***

user.name=ealalin

C:\learning_log>git config --local –list

C:\learning_log>git config --system --list

 C:\learning_log>git config --local --list

core.repositoryformatversion=0

core.filemode=false

core.bare=false

core.logallrefupdates=true

core.symlinks=false

core.ignorecase=true

remote.origin.url=https://github.com/ealalin/learning_log.git

remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*

user.name=ealalin

I have review all the repository that I have , I have put them to public


The problem stay the same !!!!! If you have any advises they are the welcome .


Regards ealalin


________________________________
